### PR TITLE
COMMERCE-11291 Add functional test coverage for COMMERCE-10897

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-11291

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase [hunk 1]
```diff
@@ -1618,6 +1618,81 @@ definition {
...
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("And a product 'ABS Sensor' with Availability Estimates XSS entry is updated") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase:1647]: "And a new Availability Estimate is created using an XSS and the product 'ABS Sensor' is updated with this entry"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase [hunk 2]
```diff
@@ -1618,6 +1618,81 @@ definition {
...
+
+			Select(
+				key_title = "Availability Estimate",
+				locator1 = "CommerceEntry#CATALOGS_FORM",
+				value1 = "Approx <script>alert(123)</script> Days");
+
+			Button.clickPublish();
+		}
+
+		task ("And the warehouse quantity for the product is set to 0") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase:1672]: "And the stock quantity of the product is set to 0"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase [hunk 3]
```diff
@@ -1618,6 +1618,81 @@ definition {
...
+	@description = "This is a test for COMMERCE-10897. Users cannot view XSS with Availability Estimates in product details"
+	@priority = 3
+	test CannotExecuteXSSWithAvailabilityEstimatesInProductDetails {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a regular user and Minium site membership is created") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase:1630]: "And a regular user with Minium site membership is created"

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**